README.md

```diff
@@ -99,6 +99,46 @@ config.settings.DSGVOBanner = {
   privacy_url: '/privacy',
 };
 ```
+You can customize the Style of the Modal Buttons
+```
+config.settings.DSGVOBanner.cssClasses = {
+  bannerAgreeButton: "branded olive",
+  bannerAdjustButton: "branded blue",
+  ...(config.settings.DSGVOBanner || {}),
+}
+```
+Or something like  
+```
+config.settings.DSGVOBanner.cssClasses = {
+  bannerAgreeButton: "red",
+  bannerAdjustButton: "pink",
+  ...(config.settings.DSGVOBanner || {}),
```

Comment:
This will copy the other DSGVOBanner settings into the cssClasses object, which doesn't make any sense.
```suggestion
```